File: app/src/main/java/com/antyzero/smoksmog/ui/screen/order/dialog/AddStationDialogComponent.java
package com.antyzero.smoksmog.ui.screen.order.dialog;


import com.antyzero.smoksmog.ui.screen.SupportFragmentModule;

import dagger.Subcomponent;

@Subcomponent(
        modules = {
                SupportFragmentModule.class
        }
)
public interface AddStationDialogComponent {

    void inject( AddStationDialog addStationDialog );
}

File: app/src/main/kotlin/com/antyzero/smoksmog/ui/screen/order/dialog/AddStationDialogComponent.kt
package com.antyzero.smoksmog.ui.screen.order.dialog


import com.antyzero.smoksmog.ui.screen.SupportFragmentModule

import dagger.Subcomponent

@Subcomponent(modules = arrayOf(SupportFragmentModule::class))
interface AddStationDialogComponent {

    fun inject(addStationDialog: AddStationDialog)
}
